corehq/apps/app_manager/views/notifications.py

```diff
@@ -0,0 +1,41 @@
+import json
+import datetime
+from django.utils.translation import ugettext as _
+from ws4redis.publisher import RedisPublisher
```

Comment:
Do you think it would be worthwhile to explore moving our current websocket usage to channels now that it's an official django project? https://www.djangoproject.com/weblog/2016/sep/09/channels-adopted-official-django-project/

Obviously not actionable now or in the near future, but will be easiest now that we only have two (I think?) pages that use this